### PR TITLE
Fix some maf logic

### PR DESCRIPTION
**Pull request by mck1117**
- Remove maf decoding cache.  It isn't on a particularly hot path, so the negligable performance gains aren't worth it.  Also, chopping down to 0.2 volt increments loses so much resolution that MAF wouldn't actually work very well.
- `getMafVoltage` now gets the voltage at the maf pin
- `getRealMaf` gets the mass flow rate, in kg/hr (TODO: should we do kg/s or g/s instead?

This change makes a negligible change to compiled image size, but saves 1KB (!) of RAM.

### CODE REVIEW

File: firmware/controllers/sensors/maf.h [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * @file    maf.h
  * @brief
  *
- * by the way 2.081989116 kg/h = 1 ft�/m
+ * by the way 2.081989116 kg/h = 1 ft�/m
```
Comment (rusefi) [inline firmware/controllers/sensors/maf.h]: here we change risky but working symbol into something not working?

Comment (mck1117) [reply firmware/controllers/sensors/maf.h]: ah geez this is the same thing as the degree symbols

Comment (mck1117) [reply firmware/controllers/sensors/maf.h]: Hey look, there's even a bug filed for it on vscode!
https://github.com/Microsoft/vscode/issues/52642

Comment (mck1117) [reply firmware/controllers/sensors/maf.h]: now we change risky but working symbol in to not-risky but working two symbols

Comment (OrchardPerformance) [reply firmware/controllers/sensors/maf.h]: Could always abbreviate that to CFM, its a pretty common abbreviation in the aftermarket as its used for cylinder head flows all the time.

**Pull request merged**